### ISSUE TITLE
[torrentdownload] (testing) Test search in TorrentDownload => Found no results while trying to browse this tracker. This may be an issue with the indexer, or other indexer settings such as search freeleech only etc.

### ISSUE THREAD

**Issue by RozvM** [Jackett/Jackett#14927]
### Have you checked our Troubleshooting page for your issue?

- [X] I have checked the Troubleshooting page

### Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### Have you read our Contributing Guidelines?

- [X] I have read the Contributing Guidelines

### Environment

```markdown
Using HTTP Client: HttpWebClient2
Using FlareSolverr: http://127.0.0.1:8191/
Using proxy: Disabled
App config/log directory: C:\ProgramData\Jackett	
ThreadPool MaxThreads: 1023 workerThreads, 1000 completionPortThreads 	
Running in Docker: No
Jackett variant: CoreWindows
OS version: Microsoft Windows NT 10.0.19045.0 (64bit OS)
Environment version: 6.0.25 (C:\Users\RozvM\Desktop\Jackett\)
Starting Jackett v0.21.1431
```

### Description

1. Startup Jackett and FlareSolverr.
2. Use this configuration for indexer:
![image](https://github.com/Jackett/Jackett/assets/74600479/3288279d-2eeb-4adc-9a6a-c38a9a158668)
3. Run test.

Site is up and browsing tracker manually seems to work well:
![image](https://github.com/Jackett/Jackett/assets/74600479/727b380d-5e2a-4398-a155-f854ae7cfaa3)

### Logged Error Messages

```text
2023-12-21 15:39:54.3562 Info Test search in TorrentDownload => Found 0 releases [174ms]
2023-12-21 15:39:54.3562 Error System.Exception: Test search in TorrentDownload => Found no results while trying to browse this tracker. This may be an issue with the indexer, or other indexer settings such as search freeleech only etc.

[v0.21.1431.0] System.Exception: Test search in TorrentDownload => Found no results while trying to browse this tracker. This may be an issue with the indexer, or other indexer settings such as search freeleech only etc.
   at Jackett.Common.Services.IndexerManagerService.TestIndexer(String name) in ./Jackett.Common/Services/IndexerManagerService.cs:line 0
   at Jackett.Server.Controllers.IndexerApiController.Test() in ./Jackett.Server/Controllers/IndexerApiController.cs:line 147
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in ./Jackett.Server/Middleware/CustomExceptionHandler.cs:line 61
```

**Comment by garfield69** [comment]
resolved with 1ff1f113fed1a4b73d7c0426f5a4f02956ea415c from #6451 which will be published in about 13 hours

**Comment by garfield69** [comment]
v0.21.1437
